### PR TITLE
Introduction - edits

### PR DESCRIPTION
**Pull request by brentzundel**
Edited for clarity, line length, spacing, and to add links to terminology.

***
<a href="https://pr-preview.s3.amazonaws.com/brentzundel/vc-imp-guide/pull/26.html" title="Last updated on Aug 13, 2019, 3:16 PM UTC (d009a86)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-imp-guide/26/7d3a30a...brentzundel:d009a86.html" title="Last updated on Aug 13, 2019, 3:16 PM UTC (d009a86)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -121,22 +121,22 @@ <h2>Introduction</h2>
       <em>This section is non-normative.</em>
 
       <p>
-        Welcome dear implementer of Verifiable Credentials! We understand that reading
-        and nagivating technical specifications can be confusing at times. For this reason,
-        we would like to offer you a few resources to get you started.
+Reading and navigating technical specifications can be confusing. For this
+reason we have created this guide, which provides some examples and resources
```
Comment (TallTed) [inline index.html]: `reason we`
-> `reason, we`

File: index.html [hunk 2]
```diff
@@ -121,22 +121,22 @@ <h2>Introduction</h2>
...
+for implementing protocols which make use of <a>verifiable credentials</a>.
       </p>
       <p>
-        First, you should probably become familiarized with the official
-        <a href="https://www.w3.org/TR/verifiable-claims-use-cases/">Use Cases document</a>,
-        which offers a concise but limited collection of use cases readers should review before
-        undestanding how Verifiable Claims can best help them.
+It may be useful to become familiarized with the official
```
Comment (TallTed) [inline index.html]: `become familiarized`
-> `first familiarize yourself`

File: index.html [hunk 3]
```diff
@@ -121,22 +121,22 @@ <h2>Introduction</h2>
...
-        The <a href="https://w3c.github.io/vc-data-model/">data model specification</a> is the
-        most important result of two years of hard work done by the Verifiable Claims Working Group.
-        However, the data model itself is not sufficient if you plan to use Verifiable Claims today.
-        Additional works needs to be done on identifiers as well as on proofs, as you will be able to
-        find out in the sections below.
+The <a href="https://w3c.github.io/vc-data-model/">data model specification</a>
+contains the technical details about <a>verifiable credentials</a>. However, the
+data model itself does not specify any protocols for using <a>verifiable
+credentials</a>. Additionally, these protocols may depend on identifiers and
+proof formats which themselves require more specification.
```
Comment (TallTed) [inline index.html]:
```
data model itself does not specify any protocols for using <a>verifiable 
credentials</a>. Additionally, these protocols may depend on identifiers and
proof formats which themselves require more specification.
```
->
```
data model specification does not specify any <em>protocols</em> for using
<a>verifiable credentials</a>, nor any <em>proof formats</em> or additional
<em>identifiers</em> upon which such protocols may depend.
```

Comment (brentzundel) [reply index.html]: Thanks, fixed in [aa459c1](https://github.com/w3c/vc-imp-guide/pull/26/commits/aa459c1497ccdfa8682ca2ed8138a7b747340ff4)

File: index.html (hunk 2 above)
Comment (brentzundel) [reply index.html]: Thanks, fixed in [aa459c1](https://github.com/w3c/vc-imp-guide/pull/26/commits/aa459c1497ccdfa8682ca2ed8138a7b747340ff4)

File: index.html (hunk 1 above)
Comment (brentzundel) [reply index.html]: Thanks, fixed in [aa459c1](https://github.com/w3c/vc-imp-guide/pull/26/commits/aa459c1497ccdfa8682ca2ed8138a7b747340ff4)

File: index.html [hunk 4]
```diff
@@ -122,21 +122,21 @@ <h2>Introduction</h2>
 
       <p>
 Reading and navigating technical specifications can be confusing. For this
-reason we have created this guide, which provides some examples and resources
+reason, we have created this guide, which provides some examples and resources
 for implementing protocols which make use of <a>verifiable credentials</a>.
       </p>
       <p>
-It may be useful to become familiarized with the official
+It may be useful to firt familiarize yourself with the official
```
Comment (TallTed) [inline index.html]: `firt` -> `first`

Comment (brentzundel) [reply index.html]: hah, oops. Thanks. Fixed in [e796e9a](https://github.com/w3c/vc-imp-guide/pull/26/commits/e796e9a5b3d3cca46face05b8a6e925687ea6d4d)

File: index.html [hunk 5]
```diff
@@ -121,22 +121,22 @@ <h2>Introduction</h2>
       <em>This section is non-normative.</em>
 
       <p>
-        Welcome dear implementer of Verifiable Credentials! We understand that reading
-        and nagivating technical specifications can be confusing at times. For this reason,
-        we would like to offer you a few resources to get you started.
+Reading and navigating technical specifications can be confusing. For this
```
Comment (jricher) [inline index.html]:
```suggestion

```

File: index.html [hunk 6]
```diff
@@ -121,22 +121,22 @@ <h2>Introduction</h2>
...
+reason, we have created this guide, which provides some examples and resources
```
Comment (jricher) [inline index.html]:
```suggestion
This guide, which provides some examples and resources
```

File: index.html [hunk 7]
```diff
@@ -121,22 +121,22 @@ <h2>Introduction</h2>
...
+for implementing protocols which make use of <a>verifiable credentials</a>.
```
Comment (jricher) [inline index.html]:
```suggestion
for implementing protocols which make use of <a>verifiable credentials</a> beyond those available in the core specifications.
```

**Review by jricher — CHANGES_REQUESTED** [review]

File: index.html [hunk 8]
```diff
@@ -121,22 +121,21 @@ <h2>Introduction</h2>
       <em>This section is non-normative.</em>
 
       <p>
-        Welcome dear implementer of Verifiable Credentials! We understand that reading
-        and nagivating technical specifications can be confusing at times. For this reason,
-        we would like to offer you a few resources to get you started.
+This guide, which provides some examples and resources
```
Comment (brentzundel) [inline index.html]:
```suggestion
This guide provides some examples and resources for implementing
```

File: index.html [hunk 9]
```diff
@@ -121,22 +121,21 @@ <h2>Introduction</h2>
...
+for implementing protocols which make use of <a>verifiable credentials</a> beyond those available in the core specifications.
```
Comment (brentzundel) [inline index.html]:
```suggestion
protocols which make use of <a>verifiable credentials</a>, beyond those available in the core specification.
```

**Review by ken-ebert — APPROVED** [review]

**Pull request merged**